conformity/fields/meta.py

```diff
@@ -46,6 +46,18 @@ def introspect(self):
         }
 
 
+class Null(Base):
+    introspect_type = 'null'
+
+    def errors(self, value):
+        if value is not None:
+            return [Error('Value is not None')]
```

Comment:
`s/None/null/` to keep it consistent with the class name? (which also will make more sense if it's returned to a client who got there via JSON)

Reply:
Good call. Changed.